tsrc/cli/push.py

```diff
@@ -129,9 +132,24 @@ def execute(self):
         self.post_push()
 
 
-def main(args):
+@click.command("push")
+@click.option("-w", "--workspace")
+@click.option("-f", "--force", is_flag=True)
+@click.option("-t", "--target", "target_branch", metavar="<target>", default="master")
+@click.option("-a", "--assignee", metavar="<assignee>")
+@click.option("--title", "mr_title", metavar="<title>", is_flag=False)
+@click.option("--accept", is_flag=True)
```

Comment:
At least the complexity is visible instantly!

Reply:
Yeah. In the argparse version we could group gitlab and github related options.

Not sure what to do here